test/src/GrumPHPTest/Util/PhpVersionTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace GrumPHPTest\Util;
+
+use GrumPHP\Util\PhpVersion;
+
+/**
+ * Class PhpVersionTest
+ * @package GrumPHPTest\Util
+ */
+class PhpVersionTest extends \PHPUnit_Framework_TestCase
```

Comment:
Is there a reason why u added a phpunit test instead of a phpspec test?
This logic is perfectly testable with phpspec. I only want to use phpunit for more advanced integrations between multiple components. Otherwise it isn't clear when to use phpspec or phpunit.